pydantic_ai_slim/pydantic_ai/result.py

```diff
@@ -475,4 +436,9 @@ async def _usage_checking_iterator():
 
         return _usage_checking_iterator()
     else:
-        return stream_response
+
+        async def _iterator():
+            async for item in stream_response:
+                yield item
+
+        return _iterator()
```

Comment:
```suggestion
        return aiter(stream_response)
```
?

Reply:
Actually that only works in 3.10 so maybe your thing is fine, though maybe we defined our own `aiter` thing to get around this in 3.9? 🤷